### DISCUSSION TITLE
[Script request] Daloradius

### DISCUSSION THREAD

**Discussion by VVsta** [community-scripts/ProxmoxVE#526] [Request script]
### Application Name

DaloRadius

### Website

https://www.daloradius.com/

### Description

daloRADIUS is an advanced RADIUS web platform aimed at managing Hotspots and general-purpose ISP deployments. It features rich user management, graphical reporting, accounting, and integrates with GoogleMaps for geo-locating (GIS). daloRADIUS is written in PHP and JavaScript and utilizes a database abstraction layer which means that it supports many database systems, among them the popular MySQL, PostgreSQL, Sqlite, MsSQL, and many others.

It is based on a FreeRADIUS deployment with a database server serving as the backend. Among other features it implements ACLs, GoogleMaps integration for locating hotspots/access points visually and many more features. daloRADIUS is essentially a web application to manage a radius server so theoretically it can manage any radius server but specifically it manages FreeRADIUS and it's database structure. Since version 0.9-3 daloRADIUS has introduced an application-wide database abstraction layer based on PHP's PEAR::DB package which support a range of database servers.

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by MickLesk** [comment]
The last release was over 3 years ago. 
I reject the script request, the tool is too old for me, too little maintained. 

[Releases 6](https://github.com/lirantal/daloradius/releases)
1.3
Latest
on Nov 4, 2021

**Comment by VVsta** [comment]
Sorry, you can see the last releases here: https://github.com/lirantal/daloradius
